packages/frontend-shared/src/components/Select.cy.tsx

```diff
@@ -170,19 +168,19 @@ describe('<Select />', () => {
     it('renders all of the slots', () => {
       const vSlots = {
         'item-body': () => 'Item Body',
-        'item-prefix': () => <Icon icon={IconHeart} data-testid="item-prefix" />,
-        'item-suffix': () => <Icon icon={IconHeart} data-testid="item-suffix" />,
+        'item-prefix': () => <div data-testid="item-prefix"></div>,
+        'item-suffix': () => <div data-testid="item-suffix"></div>,
         'selected': () => 'Selected',
-        'input-prefix': () => <Icon icon={IconHeart} data-testid="input-prefix" />,
-        'input-suffix': () => <Icon icon={IconHeart} data-testid="input-suffix" />,
+        'input-prefix': () => <div data-testid="input-prefix"></div>,
+        'input-suffix': () => <div data-testid="input-suffix"></div>,
```

Comment:
```suggestion
        'input-prefix': () => <div data-testid="input-prefix">np</div>,
        'input-suffix': () => <div data-testid="input-suffix">ns</div>,
```

Reply:
I didn't do that because the findByText isn't partial. At first I swapped it with an emoji, but the tests started failing because it was looking for `'Selected' === '❤️Selected❤️'` to match

Reply:
I agree with your comment though.